resources/services/order.ts

```diff
@@ -1,28 +1,11 @@
 import axios from 'axios'
-import {OrderItemStateType} from '../models/order'
+import {OrderItemFormDataType} from '../models/order'
 import {orderRoute} from '.'
 
-export const addOrder = (payload: OrderItemStateType) => axios({
-    url: orderRoute,
-    method: 'POST',
-    data: payload,
-})
-
 export const getOrder = () => axios.get(orderRoute)
 
-export const filterPurchaseOrder = (payload: {}) => axios({
-    method: 'GET',
-    url: orderRoute,
-    params: payload
-})
-
-export const editOrder = (payload: any) => axios({
+export const addOrder = (payload: OrderItemFormDataType) => axios({
+    url: orderRoute.concat(`/orderables/${0}`),
```

Comment:
${0}?

Reply:
Since the orderable id needs to be specified, 0 signifies adding instead of updating an orderable.